core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -153,6 +153,7 @@
 
 /** Unit tests for {@link ManagedChannelImpl}. */
 @RunWith(JUnit4.class)
+@SuppressWarnings("deprecation")
```

Comment:
Because this test is going to be deleted along with `ManagedChannelImpl`. It's better to be annotated `@Deprecated`.